frontend/src/styles.css

```diff
@@ -0,0 +1,4 @@
+/* You can add global styles to this file, and also import other style files */
```

Comment:
Avoid comments with personal pronouns. You can reword as 
/* Defines global styles and imports other style files. */